python_modules/libraries/dagster-sdf/dagster_sdf/__init__.py

```diff
@@ -6,9 +6,9 @@
     dagster_name_fn as dagster_name_fn,
     default_asset_key_fn as default_asset_key_fn,
 )
-from .information_schema import SdfInformationSchema as SdfInformationSchema
 from .resource import SdfCliResource as SdfCliResource
+from .sdf_information_schema import SdfInformationSchema as SdfInformationSchema
+from .sdf_workspace import SdfWorkspace as SdfWorkspace
```

Comment:
Appears to have changes included in https://github.com/dagster-io/dagster/pull/23238/files